test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -880,7 +962,10 @@ foo``u{2195}abc
             @{ Script = "500sgb" }
             @{ Script = "10000usgb" }
             @{ Script = "10000.0usgb" }
+            @{ Script = "1uykb" }
+            @{ Script = "10_000ul" }
         )
+
         It "<Script> should throw an error" -TestCases $testInvalidNumerals {
             param($Script)
              {[ScriptBlock]::Create($Script).Invoke()} | Should -Throw
```

Comment:
We could check ErrorId with new Pester vertion.

Reply:
Sorted!